### PR TITLE
F indigo devel port grasps

### PR DESCRIPTION
**Pull request by markpitchless**
Basic port of the grasp node from GSC stack. Code basically un changed just fixed build files and moved with rename the needed msgs and actions from sr_grasp_msg into sr_robot_msgs.

### CODE REVIEW

File: sr_grasp/scripts/grasp [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+        # Action server for running this node
+        self.grasp_server = actionlib.SimpleActionServer(
+                'grasp', GraspAction, auto_start = False)
+        self.grasp_server.register_goal_callback(self.grasp_goal_cb)
+        self.grasp_server.register_preempt_callback(self.grasp_preempt_cb)
+        self.grasp_server.start()
+
+        rospy.loginfo("Looking for tactile sensors")
+        # KCL style tactile sensor
+        self.sub_tactile_kcl = {}
```
Comment (ugocupcic) [inline sr_grasp/scripts/grasp]: do we have to keep kcl in those names?

File: sr_grasp/scripts/grasp [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+            #'lf': 'finger5/ContactState',
+            'rf': 'rfdistal/ContactState',
+            'th': 'thdistal/ContactState',
+            'ff': 'ffdistal/ContactState',
+            'mf': 'mfdistal/ContactState',
+            'lf': 'lfdistal/ContactState',
+        }
+        for finger, topic in finger_map.items():
+            try:
+                rospy.wait_for_message(topic, KCL_ContactStateStamped, timeout = 0.2)
```
Comment (ugocupcic) [inline sr_grasp/scripts/grasp]: Same remark here

File: sr_grasp/scripts/grasp_planner [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            joints = {
+                'FFJ1': 1.0, 'FFJ2':1.0, 'FFJ3': 1.4, 'FFJ4': 0.0,
+                'MFJ1': 1.0, 'MFJ2':1.0, 'MFJ3': 1.4, 'MFJ4': 0.0,    
+                'RFJ1': 1.0, 'RFJ2':1.0, 'RFJ3': 1.4, 'RFJ4': 0.0,
+                'LFJ1': 1.0, 'LFJ2':1.0, 'LFJ3': 1.4, 'LFJ4': 0.0, 'LFJ5': 0.0,
+                'THJ1': 1.0365, 'THJ2': 0.6772, 'THJ3': 0.1849, 'THJ4': 1.2196, 'THJ5': 0.0579,
+            },
+            pre_joints = {'THJ4': 1.2296, 'THJ5': -0.4779},
+        )
+        power_grasp.grasp_pose.pose.position.x = 0.02
```
Comment (ugocupcic) [inline sr_grasp/scripts/grasp_planner:80]: loading from yaml is done in #187 ?

File: sr_grasp/scripts/quick_grasp [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        client.wait_for_result(rospy.Duration.from_sec(20.0))
+        if client.get_state() == GoalStatus.SUCCEEDED:
+            rospy.loginfo("Success!")
+        else:
+            rospy.logerr("Fail!")
+
+if __name__ == '__main__':
+    rospy.init_node("quick_grasp")
+    node = QuickGraspNode()
+    while True:
```
Comment (ugocupcic) [inline sr_grasp/scripts/quick_grasp:60]: have you shown this to @adelarosat and juan?

File: sr_grasp/src/sr_grasp/utils.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        'WRJ1', 'WRJ2']
+
+def _fix_j0(joints):
+    """
+    Convert joints targets using J1 and J2 to J0, which the controllers use.
+    Useful if using joint_states to get grasp joint positions.
+    """
+    for finger in ['FFJ', 'LFJ', 'MFJ', 'RFJ']:
+        if finger+'1' in joints and finger+'2' in joints:
+            joints[finger+'0'] = joints[finger+'1'] + joints[finger+'2']
```
Comment (ugocupcic) [inline sr_grasp/src/sr_grasp/utils.py:48]: still needed even with new traj ctrl?

File: sr_grasp/scripts/grasp_planner (hunk 3 above)
Comment (markpitchless) [inline sr_grasp/scripts/grasp_planner:80]: Yes

File: sr_grasp/scripts/quick_grasp (hunk 4 above)
Comment (markpitchless) [inline sr_grasp/scripts/quick_grasp:60]: Not yet, but Rob has been iusing it. I will.

File: sr_grasp/src/sr_grasp/utils.py (hunk 5 above)
Comment (markpitchless) [inline sr_grasp/src/sr_grasp/utils.py:48]: Not using the traj controller atm, so leaving in for now. It wont be later.
Going to do traj control as a seperate pr if time.

**Pull request merged**